components/time-picker/nz-time-picker.component.html

```diff
@@ -6,7 +6,8 @@
   [placeholder]="nzPlaceHolder || ('TimePicker.placeholder' | nzI18n)"
   [(ngModel)]="value"
   readonly="readonly"
-  (click)="open()">
+  (click)="open()"
+  (keyup.enter)="open()">
```

Comment:
You should add the tests about your changes.

Reply:
Added tests.